resources/views/pages/ministry/index.blade.php

```diff
@@ -67,9 +67,13 @@
             class="col-lg-3 col-md-6 col-sm-12 ministry-cards d-flex" 
             style="cursor:pointer"
           >
-            <div class="cont-1 d-flex flex-column justify-content-center">
+            <div class="cont-1 d-flex flex-column justify-content-center py-0">
               <chart label="myVueChart" 
```

Comment:
Please change `myVueChart` to something else. E.g, 2016 budget.

But it should be dynamic

Reply:
You might also consider making the years dynamic based on the data provided. But this should work for now

Reply:
@goodhands I have changed the label. The chart now reads 5-year Trend. I think that describes aptly what that chart is doing. This complements the sub-heading above which already states Ministries and amount spent